boilerplate/app/components/switch/switch.tsx

```diff
@@ -47,8 +47,16 @@ const THUMB: ViewStyle = {
 
 const makeAnimatedValue = (switchOn) => new Animated.Value(switchOn ? 1 : 0)
 
-export function Switch(props: SwitchProps) {
```

Comment:
@yulolimum I know you're re-doing Switch, so you'll have to make sure this works in the new version. Basically i just wanted to able to pass any additional accessibility props to Switch, so i extended the props type to included all ViewProps, and then did named deconstruction so i could pass a `...rest` into the main wrapper view.